finagle-mysql/src/main/scala/com/twitter/finagle/mysql/CanBeParameter.scala

```diff
@@ -71,6 +71,14 @@ object CanBeParameter {
     }
   }
 
+  implicit val bigIntCanBeParameter = {
+    new CanBeParameter[BigInt] {
+      def sizeOf(param: BigInt) = 8
+      def typeCode(param: BigInt) = Type.LongLong
+      def write(writer: MysqlBufWriter, param: BigInt) = writer.writeBytes(param.toByteArray)
```

Comment:
Are you suggesting checking the size of the array, to be sure that it is shorter than or equal to 8?